Apps/WebClient/src/ClientApp/src/components/timeline/filters.vue

```diff
@@ -86,6 +90,11 @@ export default class FilterComponent extends Vue {
         this.isVisible = false;
     }
 
+    @Watch("isListView")
+    private onIsListView() {
+        !this.isListView ? (this.showSlider = false) : (this.showSlider = true);
```

Comment:
this looks odd, how about:
```
this.showSlider =  this.isListView;
```
and if that is the case, why is the variable showSlider needed?

Reply:
good point! will change this